meta-avocado/recipes-avocado/images/avocado-pkg-sdk.bb

```diff
@@ -3,4 +3,5 @@ LICENSE = "Apache-2.0"
 
 inherit image-packages-only
 
-IMAGE_INSTALL = "packagegroup-avocado-sdk"
+SDK_PKG_EXTRA_INSTALL ??= ""
+IMAGE_INSTALL = "packagegroup-avocado-sdk ${SDK_PKG_EXTRA_INSTALL}"
```

Comment:
Delete `${SDK_PKG_EXTRA_INSTALL}`? I guess the variable gets populated in another layer.

Reply:
See other comments.